src/modules/job-info/allow.c

```diff
@@ -47,7 +47,7 @@ static int eventlog_get_userid (struct info_ctx *ctx, const char *s,
         goto error;
     }
     if (strcmp (name, "submit") != 0 || !context) {
-        flux_log_error (ctx->h, "%s: invalid event", __FUNCTION__);
+        flux_log (ctx->h, LOG_ERR, "%s: invalid event", __FUNCTION__);
```

Comment:
optional suggestion, add actual event name for easier debugging if this (highly unlikely) error is logged, i.e. `invalid event <name>`